From 539a8fb6b375b4c8edeba28f6588028123f22f69 Mon Sep 17 00:00:00 2001
From: Sal Quintanilla <sal.quintanilla@windriver.com>
Date: Sun, 7 Jul 2013 17:38:07 -0700
Subject: Enable USB Volume Change Support

Removed DEVICE_OUT_USB_DEVICE from the fixed volume devices list.

This patch should also be verified on mobile-type devices,
before being considered for upstreaming.

Category: Device Enablement
Domain: IO
Upstream-Candidate: no

Issue: AXIA-3210
Change-Id: I2d28b6b7b096dafe022799d1fc3448a255733c2a
Signed-off-by: Sal Quintanilla <sal.quintanilla@windriver.com>
---
 media/java/android/media/AudioService.java |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/media/java/android/media/AudioService.java b/media/java/android/media/AudioService.java
index b836f50..db32ee9 100644
--- a/media/java/android/media/AudioService.java
+++ b/media/java/android/media/AudioService.java
@@ -428,8 +428,7 @@ public class AudioService extends IAudioService.Stub {
     // Devices for which the volume is fixed and VolumePanel slider should be disabled
     final int mFixedVolumeDevices = AudioSystem.DEVICE_OUT_AUX_DIGITAL |
             AudioSystem.DEVICE_OUT_DGTL_DOCK_HEADSET |
-            AudioSystem.DEVICE_OUT_ANLG_DOCK_HEADSET |
-            AudioSystem.DEVICE_OUT_ALL_USB;
+            AudioSystem.DEVICE_OUT_ANLG_DOCK_HEADSET;
 
     // TODO merge orientation and rotation
     private final boolean mMonitorOrientation;
-- 
1.7.4.1

